### PR TITLE
netty: improve readability

### PR DESCRIPTION
**Pull request by steverao**
Modifying some definitions improves readability

### CODE REVIEW

**Comment by ran-su** [comment]
You may want to fix the style issues. The details can be found here: https://travis-ci.org/grpc/grpc-java/jobs/605926588#L2345

**Comment by creamsoup** [comment]
FYI, you can run following command to check the style locally
`./gradlew checkStyleMain checkStyleTest`

**Comment by steverao** [comment]
ok, thanks @ran-su @creamsoup

File: netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java [hunk 1]
```diff
@@ -64,8 +64,10 @@
 @CanIgnoreReturnValue
 public final class NettyChannelBuilder
     extends AbstractManagedChannelImplBuilder<NettyChannelBuilder> {
-
-  public static final int DEFAULT_FLOW_CONTROL_WINDOW = 1048576; // 1MiB
+  /**
+   * 1MiB.
+   */
+  public static final int DEFAULT_FLOW_CONTROL_WINDOW = 1024 * 1024;
```
Comment (zhangkun83) [inline netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java]: This comment format is meant for javadocs, while the text doesn't fit into javadocs. The original format seems more appropriate for an explanatory comment.

ditto for the other places.

Comment (steverao) [reply netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java]: Thanks for your suggestions, i got it. @zhangkun83

**Pull request merged**